content/release-10-18-0/edge-10-18-0-bundle/10_18_0_1.md

```diff
@@ -0,0 +1,28 @@
+---
+weight: 59
+title: Release 10.18.0.1
+layout: redirect
+---
+
+The release notes for **Cumulocity IoT Edge release 10.18.0.1** contain important information about all the changes, including new features, enhancements, and fixes, that have been made to both **Cumulocity IoT Edge Appliance VM** and **Cumulocity IoT Edge on Kubernetes**. If you want to know more, you can check out the [What´s New](/release-10-18-0/whatsnew-10-18-0/) section and watch our video [What's new in Cumulocity IoT 10.18](https://www.youtube.com/watch?v=pTouaQbc6SI&list=PLexm-BA6Wrr8AYnu97vpIR7uicD2wsyms).
+
+Additionally, the release notes contain [announcements](/release-10-18-0/edge-10-18-0/#announcements) regarding planned and implemented deprecations that might require some action on your part at some point.
```

Comment:
There are no announcements for Edge at this place, not sure if we should mention this here. 

Also, the link to the WhatsNew section and video above is a bit misleading for the same reason. I would actually remove both sentences.